src/Aquifer.Public.API/Endpoints/Bibles/List/Validator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FastEndpoints;
+using FluentValidation;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.List;
+
+public class Validator : Validator<Request>
+{
+    public Validator()
+    {
+        RuleFor(x => x.LanguageId)
+            .Empty()
+            .When(x => x.LanguageCode is not null)
+            .WithMessage(x => "'{PropertyName}' must be empty when 'Language Code' is specified.");
```

Comment:
Opinionated rant: I'm not a fan of fluent validation's property name `ToString()` algorithm:
1. It inserts spaces between what it thinks are word boundaries based upon casing.
2. It doesn't have an algorithm to make the new words presentable for public consumption instead of just splitting dev names.  For example, `LanguageId` becomes `'Langage Id'` instead of `'Language ID'`.
3. The default message doesn't do what I want because it doesn't capture the `When` condition, resulting in a potentially confusing error message, so I have to write my own message, anyway.
4. Using `WithMessage` and reaching into the [string format templating args](https://docs.fluentvalidation.net/en/latest/built-in-validators.html) requires looking at the fluent validation docs and make it harder for devs to know what property is meant without cross-referencing the rule lambda in their heads.
5. I couldn't find a way to reach into this naming convention if the property I want to include in the error message is not mentioned in the baked in templating like `{PropertyName}` (hence my having to explicitly reference `'Language Code'` instead of something like `$"{ValidatorOptions.Global.DisplayNameResolver.GetPropertyName(x => x.LanguageCode)}"`).

I'd prefer it just keep the name as the API portrays it (or even better use `languageId` and convert to camel casing just like the JSON or query string presentation).  It's also possible to [globally overwrite the property name algorithm](https://docs.fluentvalidation.net/en/latest/configuring.html#overriding-the-property-name) if we want and then we could inject our own algorithm that we could also reach into for injecting property names that fluent validation doesn't provide via its built-in templating.

Nate, why do you care so much?  Because having clear, consistent error messages is very helpful for consumers of a public API.

Reply:
I could see the `{PropertyName}` stuff being more beneficial if we factored out some of our validation handling to a util somewhere. In that case yeah maybe it would make sense to have a custom algorithm?

Reply:
Can you simplify this with pattern matching? See `MachineTranslation/Update/Validator.cs`

Reply:
I don't see pattern matching helping with the property naming concern in the error message but it definitely will come in useful for `Must()` validation.